sdk/search/search-documents/src/serviceModels.ts

```diff
@@ -324,6 +327,54 @@ export interface PatternAnalyzer {
  */
 export type Analyzer = CustomAnalyzer | PatternAnalyzer | StandardAnalyzer | StopAnalyzer;
 
+/**
+ * A skill to split a string into chunks of text.
+ */
+export interface SplitSkill {
```

Comment:
this redeclares the interface, but if we go this route, won't we have to convert it in the convenience layer?

My preference would be to do a swagger translation of the x-ms-clientname of the field so we just regenerate the service client and let it pass through.

Reply:
@xirzec But such a change would impact all the languages. Would that be ok?

Reply:
If we take this as is, how will this property get mapped correctly in the generated layer?